score.py

```diff
@@ -128,6 +134,75 @@ def run_inference(images_with_header: List[ImageWithHeader],
     return segmentation
 
 
+def extract_zipped_dicom_series(zip_file_path: Path, model_folder: Path) -> List[Path]:
+    """
+    Unzip a zip file. If it contains multiple directories, assume each is a DICOM series.
+    Otherwise assume that it is a single DICOM series.
+    Return a list of DICOM series folders.
+
+    :param zip_file_path: Path to zip file.
+    :param model_folder: Path to model folder.
+    :return: List of DICOM series folders.
+    """
+    dicom_folders: List[Path] = []
+    extraction_folder = model_folder / "temp_extraction"
+    with zipfile.ZipFile(zip_file_path, 'r') as zip_file:
+        for zipped_file in zip_file.namelist():
+            if zipped_file.endswith('.dcm'):
```

Comment:
dicom files do not need to have any extension.

Reply:
I do think there should be some sort of sanity/safety check, so I've changed this to
if zipped_file.endswith('.dcm') or '.' not in zipped_file: